packages/fxa-profile-server/lib/config.js

```diff
@@ -186,6 +186,14 @@ const conf = convict({
       default: '3306',
       env: 'MYSQL_PORT',
     },
+    connectionLimit: {
+      default: 10,
+      env: 'MYSQL_CONNECTION_LIMIT',
+    },
+    queueLimit: {
+      default: 0,
+      env: 'MYSQL_QUEUE_LIMIT',
+    },
```

Comment:
I don't think these are used anywhere in the profile server?  Not sure how this patch is going affect the service when these values aren't used.  😕

Reply:
They should get passed all the way into https://github.com/mozilla/fxa/blob/main/packages/fxa-profile-server/lib/db/mysql/index.js#L29 right?

Reply:
Never mind.  The entire `mysql` config object is passed so searching for these individual configs won't turn up anything.